packages/transaction-controller/src/TransactionController.ts

```diff
@@ -2525,15 +2528,15 @@ export class TransactionController extends BaseControllerV1<
     transactionMeta: TransactionMeta,
     signedTx: TypedTransaction,
   ): Promise<void> {
-    if (signedTx.r) {
+    if (signedTx.r !== undefined) {
```

Comment:
Could signedTx.r be null?

Reply:
The `TypedTransaction` says no, but harmless to add just in case. Done.